tests/Chat.IntegrationTests/ChatActivityTest.cs

```diff
@@ -13,7 +13,7 @@ public class ChatActivityTest : AppHostTestBase
     [Fact]
     public async Task BasicTest()
     {
-        using var appHost = await TestHostFactory.NewAppHost();
+        using var appHost = await TestHostFactory.NewAppHost(Out);
```

Comment:
What if we add NewAppHost instance method to AppHostTestBase that capture Out internally, then we would be able to use shorter syntax in tests:
```
using var appHost = await NewAppHost();
```

Reply:
done